__tests__/api.test.js

```diff
@@ -31,3 +31,13 @@ describe('GET/api/topics', () => {
         });
     });
 });
+
+
+describe('GET/api', () => {
+    it('200: responds with all endpoints available', () => {
+        return request(app).get('/api')
+        .then((response) => {
+        expect(response.status).toBe(200);
+        })   
```

Comment:
Please add a test to check that the '/api' endpoint returns the endpoints data as expected. This should be a JSON describing all the available endpoints. Think about how you can retrieve the object from the endpoint.json file in your test file and use this to compare to your response body.